### PR TITLE
Simple HTML & CSS project

### PR DESCRIPTION
**Pull request by pratap-panabaka**
Simple HTML & CSS Project with "Hello World!" Text.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,1789 @@
+
```
Comment (vechicin) [inline .github/workflows/linters.yml]: You did not set up your linters correctly. Please follow [this documentation](https://github.com/microverseinc/linters-config/blob/master/html-css/README.md) to set them up correctly.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (vechicin) [inline .gitignore:1]: Your `.gitignore` file is missing the `test.md` file, please add it.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- html
+- css
+
+## Getting Started
+
+This is a startup project for "Set up a "Hello Microverse" project.
+
+## Authors
+
+PRATAP
```
Comment (vechicin) [inline README.md:16]: Please follow the README template and include the links to your professional social media profiles.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.my-color {
+	background-color: cyan;
+	color: blue; }
```
Comment (vechicin) [inline style.css]: [OPTIONAL] It is considered bad practice to use color names, consider changing them to their `hex` or `rgb` equivalents.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @PRATAP-KUMAR ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️

- [ ] Please change your PR title to something more descriptive and add a PR description

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
```
Comment (algerina) [inline .github/workflows/linters.yml:17]: Kindly, set up your linters correctly as mentioned by the previous reviewer according to the [documentation](https://github.com/microverseinc/linters-config/blob/master/html-css/README.md). You have 2 files missing` .hintrc`  and `.stylelintrc.json`

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @PRATAP-KUMAR , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- You have a meaningful PR title and description.
- Descriptive commits
- You did well implementing most of the previews TSE required changes.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @PRATAP-KUMAR 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Code Review by @omarsalem7

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**